grpc/proto/v1/generative.proto

```diff
@@ -120,6 +121,15 @@ message GenerativeOpenAI{
   optional bool is_azure = 13;
 }
 
+message GenerativeZhipuAI{
+  optional int64 max_tokens = 2;
+  string model = 3;
+  optional TextArray stop = 6;
+  optional double temperature = 7;
+  optional double top_p = 8;
+  optional string base_url = 9;
```

Comment:
as this is a new gRPC message it would be great to have the field numbers properly incremented starting from 1